integration/kubernetes/k8s-shared-volume.bats

```diff
@@ -30,6 +30,20 @@ setup() {
 	kubectl exec "$pod_name" -c "$first_container_name" -- sh -c "$cmd" | grep "$msg"
 }
 
+@test "initContainer with shared volume" {
+	pod_name="initcontainer-shared-volume"
+	last_container="last"
+
+	# Create pod
+	kubectl create -f "${pod_config_dir}/initContainer-shared-volume.yaml"
+
+	# Check pods
+	kubectl wait --for=condition=Ready pod "$pod_name"
+
+	cmd='test $(cat /volume/initContainer) -lt $(cat /volume/container)'
+	kubectl exec "$pod_name" -c "$last_container" -- sh -c "$cmd"
```

Comment:
just for ref, I found today that `kubectl wait` has a `--timeout` option - but, I think it defaults to 30s - just for ref, in case we don't think that is long enough etc.

Reply:
I wonder if the default timeout should be equal to the other tests. :thinking:  https://github.com/kata-containers/tests/blob/master/Makefile#L8

Reply:
Probably makes sense - at least to try and define the default timeouts for some operations. I didn't even know we had that `TIMEOUT` in the Makefile ;-)
In the k8s based script I'm writing right now I set some timeout values at the top of the script and then use them throughout, so at least they are easier to find/bump later if we find we need.

Reply:
I think it is fine for having the default (30s) here. If a pod takes more than that to be running, there might be something going wrong.